ground/src/main/java/com/google/android/ground/ui/common/MapConfig.kt

```diff
@@ -21,5 +21,6 @@ import com.google.android.ground.ui.map.MapType
 data class MapConfig(
   val showOfflineImagery: Boolean,
   val overrideMapType: MapType? = null,
-  val disableGestures: Boolean = false
+  val disableGestures: Boolean = false,
+  val disableRotateGestures: Boolean = false,
```

Comment:
Nit: To avoid double negatives in code, consider renaming these to `allowGestures` and `allowRotateGestures`:
https://testing.googleblog.com/2023/10/improve-readability-with-positive.html

Reply:
done